extensions/terminal-suggest/scripts/pullFishBuiltins.ts

```diff
@@ -0,0 +1,298 @@
+		description: 'In an `if` block, the `else` section contains commands that execute if none of the preceding `if` or `else if` conditions were true.',
+		args: undefined
+	},
+	'end': {
+		shortDescription: 'Terminate a block of code.',
+		description: 'Conclude a block of code initiated by constructs like `if`, `switch`, `while`, `for`, or `function`.',
+		args: undefined
+	},
+	'eval': {
+		shortDescription: 'Execute arguments as a command.',
+		description: 'Concatenate all arguments into a single command and execute it. This allows for dynamic construction and execution of commands.',
+		args: 'COMMAND...'
+	},
+	'false': {
+		shortDescription: 'Return an unsuccessful result.',
+		description: 'A command that returns a non-zero exit status, indicating failure. It is often used in scripts to represent a false condition.',
+		args: undefined
+	},
+	'realpath': {
+		shortDescription: 'Resolve and print the absolute path.',
+		description: 'Convert each provided path to its absolute, canonical form by resolving symbolic links and relative path components.',
+		args: 'PATH...'
+	},
+	':': {
+		shortDescription: 'No operation command.',
+		description: 'The `:` command is a no-op (no operation) command that returns a successful (zero) exit status. It can be used as a placeholder in scripts where a command is syntactically required but no action is desired.',
+		args: undefined
+	},
+	'test': {
+		shortDescription: 'Evaluate conditional expressions.',
+		description: 'The `test` command evaluates conditional expressions and sets the exit status to 0 if the expression is true, and 1 if it is false. It supports various operators to evaluate expressions related to strings, numbers, and file attributes.',
+		args: 'EXPRESSION'
+	},
+	'true': {
+		shortDescription: 'Return a successful result.',
+		description: 'The `true` command always returns a successful (zero) exit status. It is often used in scripts and conditional statements where an unconditional success result is needed.',
+		args: undefined
+	},
+	'printf': {
+		shortDescription: 'Display formatted text.',
+		description: 'The `printf` command formats and prints text according to a specified format string. Unlike `echo`, `printf` does not append a newline unless explicitly included in the format.',
+		args: 'FORMAT [ARGUMENT...]'
+	}
+};
+
+
+async function createCommandDescriptionsCache(): Promise<void> {
+	const cachedCommandDescriptions: Map<string, { shortDescription?: string; description: string; args: string | undefined }> = new Map();
+
+	try {
+		// Get list of all builtins
+		const builtinsOutput = await execAsync('fish -c "builtin -n"').then(r => r.stdout.trim());
+		const builtins = builtinsOutput.split('\n');
+
+		console.log(`Found ${builtins.length} Fish builtin commands`);
+
+		for (const cmd of builtins) {
+			try {
+				// Get help info for each builtin
+				const helpOutput = await execAsync(`fish -c "${cmd} --help 2>&1"`).then(r => r.stdout);
```

Comment:
These are actually based on man pages too, so we could do the pandoc thing here as well. Though the `***` weirdness is a shame so maybe just plain text is better until we figure that out.